telnyx/api_resources/sim_card.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, division, print_function
+
+from telnyx.api_resources.abstract import (
+    ListableAPIResource,
+    UpdateableAPIResource,
+    nested_resource_class_methods,
+)
+
+
+@nested_resource_class_methods(
+    "activate", path="actions/activate", operations=["create"]
+)
+@nested_resource_class_methods(
+    "deactivate", path="actions/deactivate", operations=["create"]
+)
+@nested_resource_class_methods(
+    "register",
+    path="/v2/actions/register/sim_cards",
```

Comment:
This is not a nested resource method, which means it breaks the use of `nested_resource_class_methods`. However, I thought this would be a much nicer solution than having a separate 'GlobalActiones' type class to store this action instead.